src/main/java/org/cirdles/topsoil/dataset/Dataset.java

```diff
@@ -13,13 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.cirdles.topsoil.dataset;
+package org.cirdles.topsoil.data;
```

Comment:
This doesn't match the directory structure. Should be `org.cirdles.topsoil.dataset`. Check this for everything in `src/main/org/cirdles/topsoil/dataset`.